provisioning/miniprov/test/test_cdf.py

```diff
@@ -290,30 +387,40 @@ def ret_values(value: str) -> Any:
                     'MACH_ID': 'stub',
                     'MACH_2_ID': 'stub'
                 },
-
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>site>storage_set_count': 1,
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set>server_node_count': 2,
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>name': 'StorageSet-1',
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>server_nodes': ['srvnode_1', 'srvnode_2'],
-                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>durability>data': 1,
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>site>storage_set_count':
+                1,
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set>server_node_count':
+                2,
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>name':
+                'StorageSet-1',
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>server_nodes':
+                ['srvnode_1', 'srvnode_2'],
+                'cluster>92f444df-87cc-4137-b680-aab3b35d1695>storage_set[0]>durability>data':
```

Comment:
Is pooltype(sns) needed here?

Reply:
It seems like this test doesn't go through pool pool creation that's why it can be even ignored in the test.

Reply:
Ok, Got it.